web-portal/backend/src/tenant/tenant.controller.ts

```diff
@@ -0,0 +1,43 @@
+import { Controller, Get, Post, Put, Param, Query } from '@nestjs/common';
+
+import { TenantService } from './tenant.service';
+
+@Controller('tenant')
+export class TenantController {
+  constructor(private readonly tenantService: TenantService) {}
+
+  @Get('count')
+  async generateTenant(): Promise<any> {
+    console.log('This action returns a count of total tenants');
+    return this.tenantService.countAll();
+  }
```

Comment:
I'm just wondering why count is needed?

Reply:
removing this, it was temporary for dev purpose only!